### PR TITLE
[tutorial][benchmark] nnvm -> relay

### PR DESCRIPTION
**Pull request by yzhliu**
@tqchen @masahi

### CODE REVIEW

**Comment by masahi** [comment]
Shouldn't we update get_network() too?

**Comment by yzhliu** [comment]
sorry I need to verify rpc once I get a device tomorrow.

File: apps/benchmark/util.py [hunk 1]
```diff
@@ -46,38 +47,32 @@ def get_network(name, batch_size, dtype='float32'):
     output_shape = (batch_size, 1000)
 
     if name == 'mobilenet':
-        net, params = nnvm.testing.mobilenet.get_workload(batch_size=batch_size, dtype=dtype)
+        net, params = testing.mobilenet.get_workload(batch_size=batch_size, dtype=dtype)
     elif name == 'mobilenet_v2':
-        net, params = nnvm.testing.mobilenet_v2.get_workload(batch_size=batch_size, dtype=dtype)
+        net, params = testing.mobilenet_v2.get_workload(batch_size=batch_size, dtype=dtype)
```
Comment (masahi) [inline apps/benchmark/util.py]: It seems testing.mobilenet_v2.get_workload doesn't exist, so delete it

File: apps/benchmark/util.py [hunk 2]
```diff
@@ -46,38 +47,32 @@ def get_network(name, batch_size, dtype='float32'):
...
-        net, params = nnvm.testing.resnet.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
+        net, params = testing.resnet.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
     elif "vgg" in name:
         n_layer = int(name.split('-')[1])
-        net, params = nnvm.testing.vgg.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
+        net, params = testing.vgg.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
     elif "densenet" in name:
         n_layer = int(name.split('-')[1])
-        net, params = nnvm.testing.densenet.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
+        net, params = testing.densenet.get_workload(num_layers=n_layer, batch_size=batch_size, dtype=dtype)
```
Comment (masahi) [inline apps/benchmark/util.py]: num_layers -> densenet_size
Weird, but see https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/testing/densenet.py#L84

**Comment by yzhliu** [comment]
@petrex could you also help to verify the arm results?

**Comment by petrex** [comment]
@yzhliu I can help verify the result  on AMD GPU, is that what you are looking for? thx

**Comment by masahi** [comment]
I already tested it on AMD, @yzhliu maybe you mistook it for somebody else?

**Comment by yzhliu** [comment]
@masahi @petrex sorry typo - AMD gpu, as @petrex mentioned earlier. Thanks for verifying:)

**Comment by xin71** [comment]
@yzhliu Hi, I am having hard time to reproduce the results of mobile GPU. I am wondering did you test this PR w/ mobile gpu? Thanks!!!

**Comment by yzhliu** [comment]
not yet. did you run auto tvm tuning?

**Pull request merged**